<launch>
  <!-- BEGIN PIONEER 1 -->
  <group ns="pioneer1">
    <param name="goal_x" value="-2" />
    <param name="goal_y" value="0" />
    <param name="goal_theta" value="90" />
    <node pkg="multi_nav" type="send_goal" respawn="false" name="send_goal" output="screen" args="1"/>
  </group>
  <!-- END ROBOT 0 -->

  <!-- BEGIN PIONEER 2 -->
  <group ns="pioneer2">
    <param name="goal_x" value="0" />
    <param name="goal_y" value="-2" />
    <param name="goal_theta" value="0" />
    <node pkg="multi_nav" type="send_goal" respawn="false" name="send_goal" output="screen" args="2"/>
  </group>
  <!-- END ROBOT 1 -->

  <!-- BEGIN PIONEER 3 -->
  <group ns="pioneer3">
    <param name="goal_x" value="2" />
    <param name="goal_y" value="0" />
    <param name="goal_theta" value="-90" />
    <node pkg="multi_nav" type="send_goal" respawn="false" name="send_goal" output="screen" args="3"/>
  </group>
  <!-- END ROBOT 1 -->
</launch>

  

